vcpkg/users/buildsystems/msbuild-integration.md

```diff
@@ -80,6 +80,59 @@ build](/visualstudio/msbuild/customize-your-build#directorybuildprops-and-direct
 section of the official MSBuild documentation for more information on `Directory.Build.targets` and
 `Directory.Build.props`.
 
+### Pass MSBuild properties to triplets and portfiles
+
+You can pass the value of MSBuild properties to vcpkg builds as environment variables by using the
+[`SetEnv` task](/visualstudio/msbuild/setenv-task). You must set these environment variables before the
+`VcpkgTripletSelection` task.
+
+The following example shows a MSBuild project passing the value of the `MyProp` property to vcpkg as an environment variable,
+to make it usable inside triplets and portfiles.
+
+#### Example `Directory.Build.props`
+
+```xml
+<Project>
+  <PropertyGroup>
+    <VcpkgRoot>C:\dev\vcpkg\</VcpkgRoot>
+    <MyProp Condition="'$(Platform)' == 'x64'">X64_VALUE</MyProp>
+    <MyProp Condition="'$(Platform)' == 'x86'">X86_VALUE</MyProp>
```

Comment:
Should probably use Debug|Release configurations, since changing the platform also involves changing the target triplet.